opba-db/src/test/java/de/adorsys/opba/db/SqlConvertTest.java

```diff
@@ -0,0 +1,151 @@
+package de.adorsys.opba.db;
+
+import com.google.common.io.FileWriteMode;
+import com.google.common.io.Files;
+import com.google.common.io.Resources;
+import de.adorsys.opba.db.config.TestConfig;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.util.List;
+
+import static de.adorsys.opba.db.SqlConvertTest.ENABLE_SQL_CONVERSION;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+@SpringBootTest(classes = TestConfig.class)
+@EnabledIfEnvironmentVariable(named = ENABLE_SQL_CONVERSION, matches = "true")
+public class SqlConvertTest {
+    public static final String ENABLE_SQL_CONVERSION = "ENABLE_SQL_CONVERSION";
+
+    private static final String BANKS_SOURCE = "migration/migrations/banks.csv";
+    private static final String BANKS_TARGET = "src/main/resources/migration/migrations/bank_action_data.sql";
+
+    @Value("${test.bank-action-id}")
+    private Integer bankActionId;
+
+    @Value("${test.bank-sub-action-id}")
```

Comment:
It is not clear what this property means. `bank-action-generator.sub-action.start-id` is more understandable

Reply:
done